lib/router/router.js

```diff
@@ -276,7 +283,8 @@ export default class Router extends EventEmitter {
 }
 
 function getURL () {
-  return window.location.pathname + (window.location.search || '') + (window.location.hash || '')
+  const { href, origin } = window.location
```

Comment:
If we have just `#` on the URL, `window.location.hash` simply shows a empty string.
So, we can't generate the correct URL.

But this method does that.